src/main/java/com/ubc/cpsc304/hotelify/controller/dto/AdministratorResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package com.ubc.cpsc304.hotelify.controller.dto;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+/**
+ * ResponseDto for {@link com.ubc.cpsc304.hotelify.entity.Administrator}
+ * Created by Rex on 2018-11-05
+ */
+@EqualsAndHashCode(callSuper = true)
+@Data
+public class AdministratorResponseDto extends CustomerResponseDto {
```

Comment:
In this case, you could just call it `AdministratorDto` instead of creating two DTO

Reply:
I dont get it

Reply:
you have `AdministratorRequestDto ` and `AdministratorResponseDto `. which is basically the same thing. You could just use `AdministratorDto` for both situations.

Reply:
they are not the same

Reply:
but I just removed the subclassing from `AdministratorResponseDto ` since I realized you made changes to to `CustomerResponseDto `. Now it should be good

Reply:
nvm i fk up